setup.py

```diff
@@ -44,11 +44,12 @@ def read(fname):
         "Programming Language :: Python",
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
     ],
 
     # support sagemaker-inference==1.1.0 for mxnet 1.4 eia image and
-    # sagemaker-inference==1.2.2 version for mxnet 1.6 image.
-    install_requires=['sagemaker-inference>=1.1.0,<=1.2.2', 'retrying==1.3.3'],
+    # sagemaker-inference==1.3.0 version for mxnet 1.6 image.
```

Comment:
not necessarily in scope for this PR, but I find these comments confusing. is there something in 1.1.1+ that breaks EI? do we need to set an upper bound at all?

Reply:
I'm also uncertain about why separate versions of `sagemaker-inference` are needed for the 1.4 EIA image and the 1.6 image. I'll look into it and submit another PR if needed.

Reply:
Checked with @nadiaya. It's because the newer version of MMS introduced after 1.1.0 is not backwards-compatible and breaks EIA.